markdown/__meta__.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+"""
+Python Markdown
+
+A Python implementation of John Gruber's Markdown.
+
+Documentation: https://python-markdown.github.io/
+GitHub: https://github.com/Python-Markdown/markdown/
+PyPI: https://pypi.org/project/Markdown/
+
+Started by Manfred Stienstra (http://www.dwerg.net/).
+Maintained for a few years by Yuri Takhteyev (http://www.freewisdom.org).
+Currently maintained by Waylan Limberg (https://github.com/waylan),
+Dmitry Shachnev (https://github.com/mitya57) and Isaac Muse (https://github.com/facelessuser).
+
+Copyright 2007-2018 The Python Markdown Project (v. 1.7 and later)
+Copyright 2004, 2005, 2006 Yuri Takhteyev (v. 0.2-1.6b)
+Copyright 2004 Manfred Stienstra (the original version)
+
+License: BSD (see LICENSE.md for details).
+"""
+
+from pkg_resources.extern import packaging
```

Comment:
While we're working on this, might as well address #825 as well.

Reply:
I need your opinion — are you OK with adding a new dependency on [packaging](https://pypi.org/project/packaging/), or I should leave a fallback to `pkg_resources.extern.packaging`?

Reply:
My initial reaction is to leave the fallback. That's what I intended with my previous comment. 

But what about that user who doesn't have the fallback **and** doesn't have packaging installed?

Of course, this will most likely be a problem when users are installing from their OS packaging system (such as ArchLinux as reported). But then ArchLinux could include `packaging` as a dependency to their `markdown` package. After all, they created the problem for themselves.

But that doesn't help the user who installs `pkg_resources` from ArchLinux, but `markdown` from PyPI as the original reporter in #825 did.  

On second thought, this is more complicated that I was thinking. Let's leave it for a separate PR. Sorry.

Reply:
> But that doesn't help the user who installs `pkg_resources` from ArchLinux, but `markdown` from PyPI as the original reporter in #825 did.

Note that the `setuptools` package in ArchLinux depends on `packaging` — it was the point of their patching to depend on it rather than bundle it.

So unless `pkg_resources` stops bundling `packaging` or otherwise changes its private API, there will be no problem with fallback.

Anyway, I suggested another option in #825 that you may like better.

Reply:
markdown depends on pkg_resources, and pkg_resources depends on packaging. But, the existence of its extern.packaging is an implementation detail, and the only public API of pkg_resources is get_version as mentioned there.

As @mitya57 said, the only reason this undocumented functionality actually breaks on Arch Linux is because we de-vendor "packaging" so that pkg_resources depends on it rather than its internal code copy.